{
  "cve_id": "CVE-2024-9355",
  "timestamp": "2025-01-14 16:58:44",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:40:17"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:45"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-12 21:30:04"
    }
  },
  "urls": [
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:10133",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:7502",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:7550",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:8327",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:8678",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:8847",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:9551",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://access.redhat.com/security/cve/CVE-2024-9355",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2315719",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}